src/utilities/alignment/run_10x_vdj.py

```diff
@@ -0,0 +1,269 @@
+
+# valid and deprecated reference genomes
+reference_genomes = {
+    "homo": "HG38-PLUS",
+    "hg38-plus": "HG38-PLUS",
+    "homo.gencode.v30.ERCC.chrM": "homo.gencode.v30.annotation.ERCC92",
+    "mus": "MM10-PLUS",
+    "mm10-plus": "MM10-PLUS",
+    "mm10-1.2.0": "mm10-1.2.0",
+    "mus-premrna": "mm10-1.2.0-premrna",
+    "mm10-1.2.0-premrna": "mm10-1.2.0-premrna",
+    "hg19-mm10-3.0.0": "hg19-mm10-3.0.0",
+    "microcebus": "MicMur3-PLUS",
+    "gencode.vM19": "gencode.vM19",
+    "GRCh38_premrna": "GRCh38_premrna",
+    "zebrafish-plus": "danio_rerio_plus_STAR2.6.1d",
+    "GRCh38-VDJ":"refdata-cellranger-vdj-GRCh38-alts-ensembl-4.0.0",
+}
+deprecated = {
+    "homo": "hg38-plus",
+    "mus": "mm10-plus",
+    "mus-premrna": "mm10-1.2.0-premrna",
+}
+
+# other helpful constants
+CELLRANGER = "cellranger"
+S3_RETRY = 5
+
+
+def get_default_requirements():
+    return argparse.Namespace(
+        vcpus=64, memory=256000, storage=2000, ecr_image="demuxer"
+    )
+
+
+def get_parser():
+    """ Construct and return the ArgumentParser object that parses input command.
+    """
+
+    parser = argparse.ArgumentParser(
+        prog="run_10x_count.py",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        description="Run alignment jobs using 10x",
+    )
+
+    # required arguments
+    requiredNamed = parser.add_argument_group("required arguments")
+
+    requiredNamed.add_argument(
+        "--taxon",
+        required=True,
+        choices=list(reference_genomes.keys()),
+        help="Reference genome for the alignment run",
+    )
+
+    requiredNamed.add_argument(
+        "--s3_input_path", required=True, help="The folder with fastq.gz files to align"
+    )
+
+    requiredNamed.add_argument(
+        "--s3_output_path",
+        required=True,
+        help="The folder to store the alignment results",
+    )
+
+    requiredNamed.add_argument(
+        "--num_partitions",
+        type=int,
+        required=True,
+        default=10,
+        help="Number of groups to divide samples "
+        "into for the alignment run. Enter 10 as the default "
+        "value here since we don't divide a single sample",
+    )
+
+    requiredNamed.add_argument(
+        "--partition_id",
+        type=int,
+        required=True,
+        help="Index of sample group. Enter 0 as "
+        "the default value here since we only have one sample",
+    )
+
+    # optional arguments
+    parser.add_argument("--cell_count", type=int, default=3000)
+
+    parser.add_argument(
+        "--dobby",
+        action="store_true",
+        help="Use if 10x run was demuxed locally (post November 2019)",
+    )
+
+    parser.add_argument(
+        "--region",
+        default="west",
+        choices=("east", "west"),
+        help=(
+            "Region you're running jobs in."
+            " Should match the location of"
+            " the fastq.gz files"
+        ),
+    )
```

Comment:
I would just remove this option, it should always be `west` now

Reply:
I'm keeping it just in case